codegen.mts

```diff
@@ -59,6 +58,9 @@ const config: CodegenConfig = {
             unionMember: true,
             excludeTypes: ['TokenInfoPayload'],
           },
+          mappers: {
+            Activity: '../modules/activity/module.graphql.mappers#ActivityMapper', // TODO: @eddeee888 to fix server preset not detecting interface mappers
```

Comment:
Server Preset is not detecting interface mappers so I have to this manually here. This should be an easy fix so we can do this manually here for now.